src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -1045,7 +1045,8 @@ class debug_commandscript : public CommandScript
             return false;
 
         uint32 animId = atoi((char*)args);
-        handler->GetSession()->GetPlayer()->HandleEmoteCommand(animId);
+        if (Unit* unit = handler->getSelectedUnit())
+            unit->HandleEmoteCommand(animId);
```

Comment:
Oh, I didn't know. There's some code in commands that is redundant https://github.com/ForesterDev/TrinityCore/blob/3.3.5/src/server/scripts/Commands/cs_debug.cpp#L822-L824